docs/source/reference/authenticators.md

```diff
@@ -235,21 +235,19 @@ to Spawner environment:
 
 ```python
 class MyAuthenticator(Authenticator):
-    @gen.coroutine
```

Comment:
Let's make sure we still have tests with `@gen.coroutine` to see if this is a breakage forcing chagnes on all Spawners and Authenticators. If it is, I'd argue it is not worth it to drop support.

Reply:
Could you help me with this step? I figure we need to add variations a set of tests to use a mocked spawner and mocked authenticator with @gen.coroutine in various functions.

Reply:
Can do! This is part of why I didn't update these, to make sure they are covered in the tests without duplicating everything. But we can duplicate them and add a small number of dedicated tests.